packages/react/src/Hero/Hero.tsx

```diff
@@ -72,6 +73,18 @@ function HeroDescription({size = '400', weight, children}: PropsWithChildren<Her
   )
 }
 
+type HeroLabelProps = LabelProps & BaseProps<HTMLDivElement>
+
+function HeroLabel({children, ...rest}: PropsWithChildren<HeroLabelProps>) {
+  return (
+    <div>
```

Comment:
Do we need this wrapping `div`?

Reply:
Not necessary, no. I put it there mostly for semantics as adjacent elements are all block level. I can remove it 👍